transformer/transform.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/local/bin/python3.6
+import json
+import logging
+import sys
+import typing
+import uuid
+from collections import namedtuple
+
+import argparse
+
+
+"""A metadata link is all the information necessary to get metadata
+for a particular field from the field that links to it."""
+MetadataLink = namedtuple('MetadataLink', ['source_field_name', 'linked_field_name', 'link_name'])
+
+
+class Transformer:
```

Comment:
The current `transform.py`  file and `Transformer` class are VERY specific to the Gen3 data model/format, and that should be reflected in their names. Something like: `gen3_transformer.py` and `Gen3Transfomer`.